internal/querynode/query_node.go

```diff
@@ -256,8 +255,7 @@ func (node *QueryNode) Init() error {
 		log.Info("queryNode try to connect etcd success", zap.Any("MetaRootPath", Params.EtcdCfg.MetaRootPath))
 
 		cpuNum := runtime.GOMAXPROCS(0)
-		node.cgoPool, err = concurrency.NewPool(cpuNum, ants.WithPreAlloc(true),
-			ants.WithExpiryDuration(math.MaxInt64))
```

Comment:
you can't remove ants.WithExpiryDuration
because we don't pooled routine to expire